qiskit_optimization/applications/ising/graph_application.py

```diff
@@ -0,0 +1,27 @@
+from abc import abstractmethod
+
+import networkx as nx
+
+from .base_application import BaseApplication
+
+
+class GraphApplication(BaseApplication):
+    """
+    An abstract class for graph problems in optimization problems
+    """
+
+    def __init__(self, g):
+        self._g = g.copy(as_view=True)
+        self._qp = self._build_quadratic_program()
+
+    @abstractmethod
+    def plot_graph(self):
```

Comment:
should optionally allow a solution vector `x`.

Reply:
Maybe consider using `draw` here as circuits can be rendered using draw, so this would make things consistent, and this is what nx has for drawing them too. Then the result can hopefully be *drawn* not matter whether its a set or a graph etc.

PS. Don't forget to include the standard copyright text in the content here.